ExampleCode/Java/needleDrop/src/needleDrop/Main.java

```diff
@@ -0,0 +1,44 @@
+package needleDrop;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import org.json.JSONObject;
+
+public class Main {
+
+	public static void main( String[] arg) throws Exception {
+		
+		 NeedleClass needle = new NeedleClass();
+		 Scanner input = new Scanner(System.in);
+		 iSENSE isense = new iSENSE(95500, "Testing", "Jeremy");
```

Comment:
Add constants at the class level for testing.  Do not hardcode them in the function.